src/pudl/extract/ferc1.py

```diff
@@ -49,22 +49,72 @@
 and EIA 923.
 
 """
+import csv
+import importlib
 import logging
-import os.path
-import re
-import string
+import zipfile
+from pathlib import Path
 
 import dbfread
 import pandas as pd
 import sqlalchemy as sa
+from dbfread import DBF
 
 import pudl
-import pudl.constants as pc
-import pudl.workspace.datastore as datastore
+from pudl import constants as pc
+from pudl.workspace import datastore as datastore
 
 logger = logging.getLogger(__name__)
 
 
+class Ferc1Datastore(datastore.Datastore):
+    """Provide a thin interface for pulling files from the Datastore."""
+
+    def get_folder(self, year):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=6263807)